model/edition.py

```diff
@@ -331,7 +331,7 @@ def missing_coverage_from(
         return q2
 
     @classmethod
-    def sort_by_priority(self, editions):
+    def sort_by_priority(cls, editions, license_source=None):
```

Comment:
This is a class method but we were treating it as an instance method of `LicensePool` and accessing `self.data_source`. That's why the sorting didn't work correctly before.